src/__tests__/constants.ts

```diff
@@ -1,4 +1,14 @@
-import { Profile, ProfileActions, Session, ShowData } from '../types';
+import {
+    Profile,
+    ProfileActions,
+    Session,
+    ShowData,
+    SeasonDetails,
+    Episode,
+    EpisodeDetails,
+} from '../types';
```

Comment:
```suggestion
import type {
    Profile,
    ProfileActions,
    Session,
    ShowData,
    SeasonDetails,
    Episode,
    EpisodeDetails,
} from '../types';
```
I think it might be nice to start using `import type` where applicable. This will prevent types from accidentally being used as a value. This comment can apply to any other type imports. Also, I don't think this is an existing pattern in Streamability. Feel free to just create a ticket for someone to do this later on.

Reply:
#1185